pages/linux/ethtool.md

```diff
@@ -0,0 +1,7 @@
+# ethtool
+
+> Displays and modifies Network Interface Controller (NIC) parameters.
```

Comment:
```suggestion
> Display and modify Network Interface Controller (NIC) parameters.
> More information: <http://man7.org/linux/man-pages/man8/ethtool.8.html>.
```
Let's stay impersonal, and while we're at it, also add a useful "more information" link.